www/js/splash/startprefs.js

```diff
@@ -87,7 +87,7 @@ angular.module('emission.splash.startprefs', ['emission.plugin.logger',
     startprefs.isConsented = function() {
       logger.log("curr_consented = "+$rootScope.curr_consented+
             "isIntroDone = " + startprefs.isIntroDone());
-      if (startprefs.isIntroDone() && 
```

Comment:
This file also has a bunch of extraneous changes that are not relevant to any functionality like this one.